cualid/label.py

```diff
@@ -26,7 +26,7 @@ def get_barcodes(input,
                  x_start=1.9,
                  y_start=257.2):
 
-    ids = [e.strip().split('\t')[1] for e in input]
+    ids = [e.strip().split('\t')[0] for e in input]
```

Comment:
This will now check the existing IDs against the second column of whatever file is passed. I thikn what we want to do is still check the first column, but when the IDs are created have the cual-id in the first column and the UUID in the second